webcompat/static/js/lib/bugform.js

```diff
@@ -522,8 +542,13 @@ function BugForm() {
       img_url,
       ")"
     ].join("");
+    var delimeter = "[![Screenshot Description](";
```

Comment:
@miketaylr yes, I agree, if the text area never gets updated until the form gets submitted, I think that logic is obsolete but will test and verify.